docs/user-guide/kubectl/kubectl_create_clusterrolebinding.md

```diff
@@ -0,0 +1,7 @@
+This file is autogenerated, but we've stopped checking such files into the
+repository to reduce the need for rebases. Please run hack/generate-docs.sh to
+populate this file.
+
+<!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
```

Comment:
> This analytics tag shouldn't be here, I'm pretty sure?

Purely generated and all the others have this too.  See https://raw.githubusercontent.com/kubernetes/kubernetes/master/docs/user-guide/kubectl/kubectl_create_secret.md